src/lib/stores/sdk.ts

```diff
@@ -32,9 +35,12 @@ const sdkForProject = {
     health: new Health(clientProject),
     locale: new Locale(clientProject),
     project: new Project(clientProject),
+    projectApi: new ProjectApi(clientProject),
```

Comment:
why do we need this? its the same as `project` right?

Reply:
cc @Meldiron